### PR TITLE
Merge ethService refactor, housekeeper &deferred limit parity request,  and some linter changes

### PR DESCRIPTION
**Pull request by ricardoul**

### CODE REVIEW

**Comment by vitorpy** [comment]
This pull request **fixes 3 alerts** when merging 3fe136498ca87defa493b34f54692476cbd05d86 into 0f2b0feb24a6ada0a6e47c08d1619d10e3786b4f - [view on LGTM.com](https://lgtm.com/projects/g/pillarwallet/pillar-bcx-pubsub/rev/pr-775efaaf0e849b99802e5477de7a3588e42c7ad6)

**fixed alerts:**

* 2 for Missing variable declaration
* 1 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

File: src/services/redisService.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+SOFTWARE.
+*/
+/** @module redisService.js */
+
+
+require('dotenv').config();
+const logger = require('../utils/logger.js');
+const redis = require('redis');
+const bluebird = require('bluebird');
+const config = require("../config");
```
Comment (rafaelcidade) [inline src/services/redisService.js]: single quote

File: src/services/rmqServices.test.js [hunk 2]
```diff
@@ -185,7 +185,7 @@ describe('initsRMQ ', () => {
   test('expectal call done - initSubPubMQ with tranStat content type', done => {
     const AmpqConnectMock = jest.spyOn(amqp, 'connect');
     const message = { type: 'tranStat' };
-    message.checksum = rmqServices.calculateChecksum(message);
+    message.checksum = rmqServices.calculateChecksum(message, "checksumKey");
```
Comment (rafaelcidade) [inline src/services/rmqServices.test.js]: single quote

File: src/services/rmqServices.test.js [hunk 3]
```diff
@@ -219,7 +219,7 @@ describe('initsRMQ ', () => {
   test('expectal call done - initSubPubMQ with newAsset content type', done => {
     const AmpqConnectMock = jest.spyOn(amqp, 'connect');
     const message = { type: 'newAsset' };
-    message.checksum = rmqServices.calculateChecksum(message);
+    message.checksum = rmqServices.calculateChecksum(message, "checksumKey");
```
Comment (rafaelcidade) [inline src/services/rmqServices.test.js]: single quote

File: src/services/rmqServices.test.js [hunk 4]
```diff
@@ -251,7 +251,7 @@ describe('initsRMQ ', () => {
   test('expectal call done - initSubPubMQ with updateTx content type', done => {
     const AmpqConnectMock = jest.spyOn(amqp, 'connect');
     const message = { type: 'updateTx' };
-    message.checksum = rmqServices.calculateChecksum(message);
+    message.checksum = rmqServices.calculateChecksum(message, "checksumKey");
```
Comment (rafaelcidade) [inline src/services/rmqServices.test.js]: single quote

File: src/services/rmqServices.test.js [hunk 5]
```diff
@@ -284,7 +284,7 @@ describe('initsRMQ ', () => {
   test('expectal call done - initSubPubMQ with newTx content type', done => {
     const AmpqConnectMock = jest.spyOn(amqp, 'connect');
     const message = { type: 'newTx' };
-    message.checksum = rmqServices.calculateChecksum(message);
+    message.checksum = rmqServices.calculateChecksum(message, "checksumKey");
```
Comment (rafaelcidade) [inline src/services/rmqServices.test.js]: single quote

**Comment by vitorpy** [comment]
This pull request **fixes 3 alerts** when merging e3200ae9919263661a73801f9a6da1622c03878a into 0f2b0feb24a6ada0a6e47c08d1619d10e3786b4f - [view on LGTM.com](https://lgtm.com/projects/g/pillarwallet/pillar-bcx-pubsub/rev/pr-e8e2076c60910f85a586f95b2ba8a9a335a9f25c)

**fixed alerts:**

* 2 for Missing variable declaration
* 1 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Pull request merged**